lib/elixir/lib/integer.ex

```diff
@@ -131,6 +131,9 @@ defmodule Integer do
 
   Raises an error if `base` is less than 2 or more than 36.
 
+  If an integer formatted string wants to be directly converted to a integer,
+  `String.to_integer/1` or `String.to_integer/2` can be used instead.
```

Comment:
The wording at the beginning of the sentence is kind of off (in the `Float` module as well, not your fault 😛). What if we say

> If you want to convert a string-formatted integer directly to an integer, `String.to_integer/1` or `String.to_integer/2` can be used instead.

Same thing would go for the `Float` module, it would be great if you changed that as well in this PR :) Wdyt?